examples/list_pod/main.c

```diff
@@ -0,0 +1,111 @@
+#include <apiClient.h>
+#include <CoreV1API.h>
+#include <malloc.h>
+#include <stdio.h>
+#include <errno.h>
+
+#define K8S_APISERVER_BASEPATH "https://9.111.254.254:6443"
+#define K8S_TOKEN_FILE_IN_CLUSTER "/var/run/secrets/kubernetes.io/serviceaccount/token"
+#define K8S_TOKEN_BUF_SIZE 1024
+#define K8S_AUTH_KEY "Authorization"
+#define K8S_AUTH_VALUE_TEMPLATE "Bearer %s"
+
+apiClient_t *g_k8sAPIConnector;
+
+void list_pod(apiClient_t *apiClient)
+{
+    v1_pod_list_t *pod_list = NULL;
+    pod_list = CoreV1API_listNamespacedPod( apiClient ,
+                                            "default", /*namespace*/
+                                            NULL, /* pretty */
+                                            0, /* allowWatchBookmarks */
+                                            NULL, /* continue */
+                                            NULL, /* fieldSelector */
+                                            NULL, /* labelSelector */
+                                            0, /* limit */
+                                            NULL, /* resourceVersion */
+                                            0, /* timeoutSeconds */
+                                            0 /* watch */
+                    );
+    printf("return code=%ld\n", apiClient->response_code);
+    if(pod_list){
```

Comment:
nit: spaces between `if` and `(pod_list)`

Reply:
OK